comms/dht/src/test.txt

```diff
@@ -0,0 +1,18 @@
+Node 1: 80....
```

Comment:
Just a reminder to cleanup, if we want to keep it bang it into a md file in the docs folder?